### ISSUE TITLE
Fix re-initialization during global mode change

### ISSUE THREAD

**Issue by DivyaVavili** [contiv/netplugin#729]
Global fwd mode change reinitializes ofnet states. Add reinitializing steps to completely update ofnet state (including uplink states)

**Comment by kahou82** [comment]
@tiewei is it something you have fixed?

**Comment by tiewei** [comment]
no, this is a little different. @dvavili fyi, we are re-evaluating the re-initialization feature which may be removed in later release, as a result, the fwd mode and uplink states will only allowed to be managed during the starting config.
